README.md

```diff
@@ -16,9 +22,83 @@ Or install it yourself as:
 
     $ gem install system_health
 
+and then...
+
+1. create an initializer in config/initializers/system_health.rb
+
+  ```ruby
+  SystemHealth.configure do |config|
+    config.monitor_classes = [
+      SystemHealth::Monitors::BadData
+    ]
+  end
+  ```
+
+  This should define the SYSTEM_HEALTH_MONITOR_CLASSES array with all
+  your monitor classes.  There can be as many monitor classes as you wish.
+
+2. Create your monitor classes
+
+  In lib/system_health/monitors/bad_data.rb create:
+
+  ```ruby
+  module SystemHealth
+    module Monitors
+      class BadData < Base
+        def error_messages
+          ['some error message here'] if bad_data?
+        end
+
+        private
+
+        def bad_data?
+          # logic here
+        end
+      end
+    end
+  end
+  ```
+
+  each monitor should define a public instance method for `error_messages`
+  as an array.  If that array is empty then the data is good.  When the
+  data is bad, one or more error messages can be placed in that array.
+  That is it!
+
+3. Define the environment variables
+
+  For applications hosted on Heroku, this might be something like:
+
+  ```bash
+  heroku config:add SYSTEM_HEALTH_MONITOR_USERNAME=somenamehere
+  heroku config:add SYSTEM_HEALTH_MONITOR_PASSWORD=somesecrethere
+  ```
+
+  on other environments, you'll need to define these elsewhere.
+
 ## Usage
 
-TODO: Write usage instructions here
+The System Health gem exposes a single controller endpoint at:
+yourapp://system_health/monitor
```

Comment:
`yourapp/system_health/monitor`? I'm confused that `yourapp` looks like a protocol